spec/models/concerns/current_tracked_by_changeset_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe CurrentTrackedByChangeset do
+  let(:stop) { create(:stop) }
+
+  context '#as_change' do
+    it 'converts to camelCase' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.